spec/features/idv/doc_auth/hybrid_handoff_spec.rb

```diff
@@ -11,14 +11,21 @@
     IdentityConfig.store.idv_send_link_attempt_window_in_minutes
   end
   let(:facial_match_required) { false }
+  let(:acr_values) do
+    facial_match_required ?
+      Saml::Idp::Constants::IAL_VERIFIED_FACIAL_MATCH_REQUIRED_ACR :
+      Saml::Idp::Constants::IAL_VERIFIED_ACR
+  end
```

Comment:
would it be easier to just override `acr_values` in each spot instead of `facial_match_required`?